src/Datadog.Trace.ClrProfiler.Managed/Integrations/Elasticsearch.Net/Pipeline.cs

```diff
@@ -99,7 +100,7 @@ private static Scope CreateScope(dynamic pipeline, dynamic requestData)
             string method = null;
             try
             {
-                method = requestData?.Method;
+                method = requestData?.Method?.ToString();
```

Comment:
Wouldn't it make sense to have add something like `Debug.Assert(false, "This code should be executed")` in all these empty catch closes?
Also I suppose the integration tests should be able to assert on the value of method.

Reply:
Yes, I need to go back and add tons of logging everywhere in the integrations using `ILog` like the manual instrumentation library. I just created [this Trello ticket](https://trello.com/c/R4Nv8S1G) to follow up on this.